pkg/taskservice/mysql_task_storage.go

```diff
@@ -0,0 +1,580 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package taskservice
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	_ "github.com/go-sql-driver/mysql"
+
+	"github.com/matrixorigin/matrixone/pkg/pb/task"
+)
+
+var (
+	createSqls = []string{
+		`create table if not exists sys_async_task (
+			task_id                     int primary key auto_increment,
+			task_metadata_id            varchar(16) unique not null,
+			task_metadata_executor      int,
+			task_metadata_context       blob,
+			task_max_retry_times        int,
+			task_retry_interval         int,
```

Comment:
task options and task exec result only one field is needed to store the json string inside

Reply:
fixed.